fix else statement and add subscription to activation keys

jameerpathan111 opened the pull request.

lpramuk commented on jobs/satellite6-db-upgrade-migrate.yaml:
```diff
@@ -158,10 +158,10 @@
                         predefined-parameters: |
-                          SATELLITE_VERSION=${TO_VERSION}
-                          SERVER_HOSTNAME=${SATELLITE_HOSTNAME}
+                          SATELLITE_VERSION=${{TO_VERSION}}
```
no need to change it, it's ```job:``` (only for ```job-template:``` double curved brackets are required)

lpramuk replied:
Just check https://github.com/SatelliteQE/robottelo-ci/blob/master/jobs/satellite6-installer.yaml#L237 (satellite6-installer is job (not job-template)

lpramuk commented:
Nice once given ack cannot be shaken off ;-)

jameerpathan111 commented:
> Please do not add double {{ }} to job as it's required only for job-templates
> 
> Just check https://github.com/SatelliteQE/robottelo-ci/blob/master/jobs/satellite6-installer.yaml#L237
> (satellite6-installer is also a job)
Thanks for clarification. will update PR accordingly.

jameerpathan111 commented:
@lpramuk I have removed changes regarding predefined-parameters.
I have updated PR and it contains fix related to adding subscription to activation keys.

ntkathole commented:
@lpramuk bump
